chain_bridge.go

```
@@ -81,6 +81,38 @@ func (l *LndRpcChainBridge) GetBlockHash(ctx context.Context,
 	return blockHash, nil
 }
 
+// VerifyBlock returns an error if a block (with given header and height) is not
+// present on-chain. It also checks to ensure that block height corresponds to
+// the given block header.
+func (l *LndRpcChainBridge) VerifyBlock(ctx context.Context,
+	header wire.BlockHeader, height uint32) error {
+
+	// TODO(ffranr): Once we've released 0.3.0, every proof should have an
+	// assigned height. At that point, we can remove this check.
```

nit: "At that point we can return an actual error here".